JSDemos/Demos/TreeList/MultipleRowSelection/description.md

```diff
@@ -1 +1,7 @@
-In&nbsp;this demo, the TreeList component uses the multiple selection mode. To&nbsp;select a&nbsp;row, click the check box located in&nbsp;this row. Note that selection is&nbsp;non-recursive by&nbsp;default, that&nbsp;is, only the clicked row is&nbsp;selected, but not its nested rows. To&nbsp;make it&nbsp;recursive, click the _&laquo;Recursive Selection&raquo;_ check box under the TreeList. You can also select all rows at&nbsp;once by&nbsp;clicking the check box in&nbsp;the column header.
\ No newline at end of file
+This demo enables *"multiple"* **selection**.[mode](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/selection/#mode). This mode allows users to press keyboard shortcuts or click checkboxes to select multiple rows.
```

Comment:
`**selection**.[mode]`

Is the period in between the two words a typo? For example, do you mean:

`**selection** [mode]`

Reply:
No, the dot is there to denote this property: https://js.devexpress.com/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/selection/#mode.